include/grpc/grpc.h

```diff
@@ -374,7 +371,7 @@ GRPCAPI void grpc_call_destroy(grpc_call *call);
     \a grpc_server_register_completion_queue. */
 GRPCAPI grpc_call_error grpc_server_request_call(
     grpc_server *server, grpc_call **call, grpc_call_details *details,
-    grpc_metadata_array *request_metadata,
+    grpc_metadata **request_metadata, size_t *request_metadata_count,
```

Comment:
I assume the resulting request_metadata is owned by the call. Maybe add that to the doc. Also below.